service/resource/deployment/error.go

```diff
@@ -4,11 +4,11 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
-var createTimeoutError = microerror.New("create timeout")
+var timeoutError = microerror.New("timeout")
 
-// IsCreateTimeoutError asserts createTimeoutError.
-func IsCreateTimeoutError(err error) bool {
-	return microerror.Cause(err) == createTimeoutError
+// IsTimeoutError asserts createTimeoutError.
+func IsTimeoutError(err error) bool {
+	return microerror.Cause(err) == timeoutError
```

Comment:
That has nothing to do with vendoring AFAIK.

Reply:
You are right. I created more generic error, which I think is the general direction we follow.
Do you want to revert?

Reply:
No. Lets just try to do one thing in a PR. I screw this on my own quite a lot. Lets just keep trying.